### PR TITLE
email template

### PR DESCRIPTION
**Pull request by kanuj365**
Targets #2539

### Description
Resume Screening , Introductory call , Basic technical round. 

### Checklist:
- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/HR/Database/Seeders/SettingsTableSeeder.php [hunk 1]
```diff
@@ -22,7 +22,7 @@ public function run()
...
...color: #212529; font-family: Raleway, sans-serif; font-size: 14.4px; white-space: pre-wrap;"><span style="color: #000000; font-family: Verdana, Arial, Helvetica, sans-serif; font-size: 14px;">We are reviewing your application and will get back to you soon.<br /><br /></span>Meanwhile, in case you have questions, now or later, feel free to email us. We\'ll be happy to answer those for you. </p><p class="p1" style="box-sizing: border-box; margin-top: 0px; margin-bottom: 1rem; color: #212529; font-family: Raleway, sans-serif; font-size: 14.4px; white-space: pre-wrap;">Looking forward to chatting soon.</p><p class="p1" style="box-sizing: border-box; margin-top: 0px; margin-bottom: 1rem; color: #212529; font-family: Raleway, sans-serif; font-size: 14.4px; white-space: pre-wrap;">Our FB page is the best place to get most of the update about happening at ColoredCow, be it our work or opportunities to work together. Check that out to know more <a style="box-sizing: border-box; color: #007bff; text-decoration-line: none; background-color: transparent;" href="https://www.facebook.com/ColoredCowConsulting/">https://www.facebook.com/ColoredCowConsulting/</a></p>HR Teame<br />ColoredCow</div>',
```
Comment (Shivam-Samant) [inline Modules/HR/Database/Seeders/SettingsTableSeeder.php:25]: Why are you change in this file?

File: Modules/HR/Database/Seeders/HRRoundsTableSeeder.php [hunk 2]
```diff
@@ -29,7 +29,7 @@ public function run()
...
... to select a time slot and schedule your interview with ColoredCow.</br>|*interview_slot_link*|<br/><br />This will be a chance for you to learn more about ColoredCow and for us to learn more about you and your concepts, and knowledge. <br /><br />As we said earlier we are in an interesting phase of growth and offer a chance for hands-on learning and an opportunity to take your career in the direction of your dreams. <br /><br />In case you have questions, now or later, feel free to email me. I\'ll be happy to answer those for you.<br /><br />Looking forward to chatting soon.</p><p class=\\"p1\\">Our FB page is the best place to get most of the update about happening at ColoredCow, be it our work or opportunities to work together. Check that out to know more <a href=\\"https://www.facebook.com/ColoredCowConsulting/\\">https://www.facebook.com/ColoredCowConsulting/</a><a href=\\https://docs.google.com/document/d/1GfhDmoZjA5CnFqpw8Ca568QbqhyZFUU34arMkVKxqhw/edit/>https://docs.google.com/document/d/1GfhDmoZjA5CnFqpw8Ca568QbqhyZFUU34arMkVKxqhw/edit</a><br /><br />Thanks<br />HR Team<br />ColoredCow</p>", "subject": "ColoredCow Congratulates you again on making it to the next round!"}',
```
Comment (Shivam-Samant) [inline Modules/HR/Database/Seeders/HRRoundsTableSeeder.php:32]: Why are you changing the Telephonic Interview round template?

File: Modules/HR/Database/Seeders/HRRoundsTableSeeder.php [hunk 3]
```diff
@@ -29,7 +29,7 @@ public function run()
...
             'in_trial_round' => 0,
         ]);
```
Comment (Shivam-Samant) [inline Modules/HR/Database/Seeders/HRRoundsTableSeeder.php:35]: You need to make the changes in the rejected template as well

Comment (kanuj365) [reply Modules/HR/Database/Seeders/HRRoundsTableSeeder.php:35]: the task's requirement.

**Review by P4NK4J — CHANGES_REQUESTED** [review]
@kanuj365 You only needed to update the content of the email template, instead of updating the HTML coding, adding JSON decode etc. This will work on your local machine but throw an error on production. Please change only the content in the templates. Also, there are different files changed here, the only file which is supposed to be updated is `HRRoundsTableSeeder.php`, please make the changes and assign for review.

**Review by P4NK4J — APPROVED** [review]
Looks good.

**Pull request merged**